podman-image-daily/Containerfile

```diff
@@ -42,6 +42,8 @@ RUN rpm-ostree override replace --experimental --freeze \
 # Install qemu-user-static for bootc
 RUN rpm-ostree install subscription-manager qemu-user-static && rm -fr /var/cache
 RUN systemctl enable rhsmcertd.service
+# Patching qemu backed binfmt configurations to use the actual executable's permissions and not the interpreter's
+RUN for x in /usr/lib/binfmt.d/*.conf; do sed 's/\(:[^C:]*\)$/\1C/' "$x" | sudo tee /etc/binfmt.d/"$(basename "$x")"; done
```

Comment:
nit: we already run as root here so no need for `sudo`

Reply:
Oops. Done.